### PR TITLE
[20.10 backport] profiles: seccomp: update to Linux 5.11 syscall list

### PR DESCRIPTION
**Pull request by thaJeztah**
backport of https://github.com/moby/moby/pull/41889

These syscalls (some of which have been in Linux for a while but were
missing from the profile) fall into a few buckets:

 * close_range(2), epoll_pwait2(2) are just extensions of existing "safe
   for everyone" syscalls.

 * The mountv2 API syscalls (fs*(2), move_mount(2), open_tree(2)) are
   all equivalent to aspects of mount(2) and thus go into the
   CAP_SYS_ADMIN category.

 * process_madvise(2) is similar to the other process_*(2) syscalls and
   thus goes in the CAP_SYS_PTRACE category.

**- Description for the changelog**

### CODE REVIEW

**Comment by thaJeztah** [comment]
@tonistiigi @AkihiroSuda ptal

**Review by AkihiroSuda — APPROVED** [review]

**Comment by thaJeztah** [comment]
rebased to trigger CI with test-fixes that were merged

File: profiles/seccomp/default.json [hunk 1]
```diff
@@ -590,9 +592,15 @@
 				"bpf",
 				"clone",
 				"fanotify_init",
+				"fsconfig",
+				"fsmount",
```
Comment (cpuguy83) [inline profiles/seccomp/default.json:596]: It seems strange we allow this one.

Comment (cpuguy83) [reply profiles/seccomp/default.json:596]: e.g. `move_mount`, given that mounts tend to leak into the container.

Comment (cpuguy83) [reply profiles/seccomp/default.json:596]: Nevermind, I see this is for CAP_SYS_ADMIN.

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Review by tiborvass — APPROVED** [review]

**Pull request merged**